### PR TITLE
Refactor commands

### LINKED ISSUE #19: Allow deleting definitions
Introduce new command: /rm definition. Should take ACL into consideration

### LINKED ISSUE #20: Change /wtf to only return definition

### LINKED ISSUE #21: Introduce /def
The command should create and update the issues. Just like /wtf right now

### PR DESCRIPTION
**Pull request by Zifius**
Fixes #19
Fixes #20
Fixes #21

Introduces #25

### CODE REVIEW

File: handlers.py [hunk 1]
```diff
@@ -30,26 +30,33 @@ def __init__(self, definition_repo):
         super().__init__()
         self.definitionRepo = definition_repo
 
+    def handle(self, bot, update):
```
Comment (alexandra-trisch) [inline handlers.py]: do we use it somewhere?

File: handlers.py [hunk 2]
```diff
@@ -62,15 +69,49 @@ def handle(self, bot, update):
...
+            reply("Please provide a term for lookup.")
+            return
+
+        term = parts[1]
+        definition = self.definitionRepo.findByTerm(term)
 
         if definition is None:
             reply("I do not have any info for the term: '{}'".format(term))
         else:
             reply(("Term '{}' means: {}".format(definition.term, definition.content)))
```
Comment (alexandra-trisch) [inline handlers.py]: why double parentheses?

Comment (Zifius) [reply handlers.py]: I thought it's Python standard. Is there a PEP for this?

Comment (alexandra-trisch) [reply handlers.py]: this line isn't really different from the one above, you pass a string into `reply`. But you pass it inside of `()`

File: handlers.py (hunk 1 above)
Comment (alexandra-trisch) [reply handlers.py]: should be fixed by #26, this function wasn't used correctly and that's why we had problems with concurrent threads

File: handlers.py (hunk 2 above)
Comment (Zifius) [reply handlers.py]: Oh sorry, I was reading "quotes" not "parentheses" all the time

File: handlers.py (hunk 1 above)
Comment (Zifius) [reply handlers.py]: Gotcha, thanks, closing my PR in favour of yours

**Comment by Zifius** [comment]
Superseded by #26

**Pull request closed without merging**